x11-wm/aewm++/aewm++-1.1.5-r2.ebuild

```diff
@@ -19,16 +19,13 @@ DEPEND="${RDEPEND}
 	x11-proto/xextproto
 	x11-proto/xproto"
 
-src_prepare() {
-	epatch "${FILESDIR}"/${P}-gcc43.patch \
-		"${FILESDIR}"/${P}-makefile.patch
-}
+PATCHES=(
+	"${FILESDIR}"/${P}-gcc43.patch
+	"${FILESDIR}"/${P}-makefile.patch
+	)
+
+DOCS=( ChangeLog README )
```

Comment:
ChangeLog and README are installed by default. So this is not needed.

Reply:
thanks for the hint, fixed it